javascript-light-bulb/main.js

```diff
@@ -0,0 +1,17 @@
+var clicked = 0;
+
+var $switch = document.querySelector('.button');
+var $backGround = document.querySelector('.background');
+
+function lightBulb(event) {
+  clicked++;
+  if (clicked % 2 === 0) {
```

Comment:
You aren't using the click count here for anything other than checking if the value is even or odd. In this case, a boolean value would be a better way of keeping track of if the light is currently on or not.